WickedEngine/shaders/skyPS_dynamic.hlsl

```diff
@@ -3,20 +3,13 @@
 
 float4 main(float4 pos : SV_POSITION, float2 clipspace : TEXCOORD) : SV_TARGET
 {
-	float4 color = 0;
-	
-	// When realistic sky 'High Quality' is enabled, let's disable skybox color since we render high quality sky in SkyAtmosphere
-	[branch]
-	if ((GetFrame().options & OPTION_BIT_REALISTIC_SKY_HIGH_QUALITY) == 0)
-	{
-		float4 unprojected = mul(GetCamera().inverse_view_projection, float4(clipspace, 0.0f, 1.0f));
-		unprojected.xyz /= unprojected.w;
+	float4 unprojected = mul(GetCamera().inverse_view_projection, float4(clipspace, 0.0f, 1.0f));
+	unprojected.xyz /= unprojected.w;
 
-		const float3 V = normalize(unprojected.xyz - GetCamera().position);
+	const float3 V = normalize(unprojected.xyz - GetCamera().position);
 		
-		color = float4(GetDynamicSkyColor(V), 1);
-		color = clamp(color, 0, 65000);
-	}
+	float4 color = float4(GetSimpleSkyColor(V), 1);
```

Comment:
We can't draw the realistic sky in the pixel shader anymore?

Reply:
Maybe I misunderstood. So we keep pixel shader, but only disable when compute is required?

Reply:
Yeah I thought keep the current rendering of realistic sky in pixel shader, but if you switch to high quality and compute shader is required, then disable the sky draw call.

Reply:
Gotcha, give me a minute

Reply:
I've pushed new changes